scripts/convert-pure-esm-to-cjs.js

```diff
@@ -0,0 +1,3 @@
+const {execSync} = require('child_process');
+
+execSync("esbuild --platform=node --bundle node_modules/import-meta-resolve/lib/resolve.js > lib/converted-esm/import-meta-resolve.js", {shell: true})
```

Comment:
This should be gone once the package can be converted to esm, right?

Reply:
Yes! Unfortunately, it can't be converted to ESM, since, if I understand correctly, ESLint doesn't yet accept ESM plugins.

Reply:
yes, but likely to be supported in eslint v9.